src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs

```diff
@@ -107,7 +106,23 @@ private void EnableRunInteractivity()
 
         private void SettingsButton_Click(object sender, RoutedEventArgs e)
         {
-            //Launch the Preference panel in the Security tab
+            const string securityTabName = "Security";
+            const string trustedTabExpanderName = "TrustedPathsExpander";
+
+            var preferencesWindow = new PreferencesView(mainWindow as DynamoView);
+            var tabControl = preferencesWindow.preferencesTabControl;
+            if (tabControl == null) return;
+            var securityTab = (from TabItem tabItem in tabControl.Items
+                               where tabItem.Header.ToString().Equals(securityTabName)
+                               select tabItem).FirstOrDefault();
+            if (securityTab == null) return;
+            tabControl.SelectedItem = securityTab;
+            var trustedPathExpander = securityTab.FindName(trustedTabExpanderName) as Expander;
+            if (trustedPathExpander == null) return;
+            trustedPathExpander.IsExpanded = true;
```

Comment:
can we wrap this into a function which take params?  I am pretty sure we have other use cases that require us opening another tab